src/app/game/game.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Observable, from, of, defer } from 'rxjs';
+
+import { GameComponent } from './game.component';
+import { Answer } from '../models/answer';
+import { AnswerService } from '../services/answer.service';
+
+const mockAnswer = { id: 1, word: 'ADEPT' } as Answer;
```

Comment:
There is absolutely nothing wrong IMO with using `as`, but in Shure's codebase we have to use <Answer> syntax, so it'd be written like:

```typescript
const mockAnswer = <Answer>{ id: 1, word: 'ADEPT' };
```

I just want to make sure you are familiar with this syntax. The linter will stop you from using `as`

Reply:
When I've used typescript in the past it's been with React using `.tsx` files so I would often use `as` to be safe. It's a helpful reminder that the `as` isn't necessary in this context 😁